cmd/amppkg_test_cache/main.go

```diff
@@ -42,6 +42,7 @@ func main() {
 	})
 	http.HandleFunc("/test.wpk", func(resp http.ResponseWriter, req *http.Request) {
 		resp.Header().Set("Content-Type", "application/signed-exchange;v=b0")
+		resp.Header().Set("Content-Disposition", "attachment")
```

Comment:
Accidental change?

Reply:
Not accidental, but temporary; I wasn't sure the AMP Cache could make this change, but it turned out to be easy. Reverted.